codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2Headers.java

```diff
@@ -354,4 +286,61 @@ public ByteString path() {
     public ByteString status() {
         return get(PseudoHeaderName.STATUS.value());
     }
+
+    @Override
+    public int hashCode() {
+        return size();
```

Comment:
@normanmaurer I agree. I only overwrote `hashCode()`, because I overrode `equals` and I only overrode equals because it's useful in tests to compare two header objects. The 4.0 implementation doesn't event override hashCode() and equals(). Take a look at those comments https://github.com/netty/netty/pull/3681/files#diff-c8a20821660fbcebfc60dea15a3a3f79R867